ProcessingStep missing from adam-codegen

Discovered as a compile error when updating #1505.

```bash
$ mvn compile
...
[ERROR] adam/adam-core/target/generated-sources/src/main/scala/org/bdgenomics/adam/sql/Schemas.scala:1001: error: not found: type ProcessingStep
[ERROR]     val processingSteps: Seq[ProcessingStep]) extends Product {
[ERROR]                              ^
[ERROR] adam/adam-core/target/generated-sources/src/main/scala/org/bdgenomics/adam/sql/Schemas.scala:957: error: not found: type ProcessingStep
[ERROR]     processingSteps: Seq[ProcessingStep]): RecordGroup = {
[ERROR]                          ^
[ERROR] adam/adam-core/target/generated-sources/src/main/scala/org/bdgenomics/adam/sql/Schemas.scala:985: error: not found: value ProcessingStep
[ERROR]       record.getProcessingSteps().toSeq.map(r => ProcessingStep.fromAvro(r)))
[ERROR]                                                  ^
[ERROR] three errors found
```

--- a/adam-core/pom.xml
+++ b/adam-core/pom.xml
@@ -109,6 +109,7 @@
                 <argument>org.bdgenomics.formats.avro.Genotype</argument>
                 <argument>org.bdgenomics.formats.avro.NucleotideContigFragment</argument>
                 <argument>org.bdgenomics.formats.avro.OntologyTerm</argument>
+                <argument>org.bdgenomics.formats.avro.ProcessingStep</argument>
                 <argument>org.bdgenomics.formats.avro.TranscriptEffect</argument>
                 <argument>org.bdgenomics.formats.avro.Variant</argument>
                 <argument>org.bdgenomics.formats.avro.VariantAnnotation</argument>

